Dockerfile

```diff
@@ -3,5 +3,10 @@ FROM busybox:ubuntu-14.04
 RUN mkdir -p /opt
 ADD ./aws-operator /opt/aws-operator
 
+RUN mkdir -p /opt/resources
```

Comment:
I don't think the file needs to be copied. The template path is `resources/templates/cloudformation/auto_scaling_group.yaml` which is the local path. In the docker image the path is `/opt/resources/templates/cloudformation/auto_scaling_group.yaml` but the work dir is set to `/opt` so the path is the same.

The binary should work if its run from the checked out directory.